hardware_interface/include/hardware_interface/joint_state_interface.h

```diff
@@ -64,18 +140,75 @@ class JointStateHandle
     {
       throw HardwareInterfaceException("Cannot create handle '" + name + "'. Effort data pointer is null.");
     }
+
   }
 
   std::string getName() const {return name_;}
   double getPosition()  const {assert(pos_); return *pos_;}
   double getVelocity()  const {assert(vel_); return *vel_;}
   double getEffort()    const {assert(eff_); return *eff_;}
 
+  const double* getPositionPtr() const {return pos_;}
+  const double* getVelocityPtr() const {return vel_;}
+  const double* getEffortPtr()   const {return eff_;}
+
+  double getAbsolutePosition() const {
+    assert(absolute_pos_);
+    if(!hasAbsolutePosition()){
+      throw std::runtime_error("Joint state handle does not contain absolute encoder position information");
+    }
```

Comment:
As in `actuator_state_interface.h`, these `assert`s should be removed

Reply:
thanks